cli/ChangePasswordCommand.php

```diff
@@ -0,0 +1,181 @@
+<?php
+namespace Grav\Plugin\Console;
+
+use Grav\Console\ConsoleCommand;
+use Grav\Common\File\CompiledYamlFile;
+use Grav\Common\User\User;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Helper\Helper;
+use Symfony\Component\Console\Question\Question;
+
+/**
+ * Class CleanCommand
+ *
+ * @package Grav\Console\Cli
+ */
+class ChangePasswordCommand extends ConsoleCommand
+{
+
+    /**
+     * @var array
+     */
+    protected $options = [];
+
+    /**
+     * Configure the command
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('change-password')
+            ->setAliases(['edit-password', 'newpass', 'changepass', 'passwd'])
+            ->addOption(
+                'user',
+                'u',
+                InputOption::VALUE_REQUIRED,
+                'The username'
+            )
+            ->addOption(
+                'password',
+                'p',
+                InputOption::VALUE_REQUIRED,
+                "The password. Note that this option is not recommended because the password will be visible by users listing the processes. You should also make sure the password respects Grav's password policy."
+            )
+            ->setDescription('Changes a User Password')
+            ->setHelp('The <info>change-password</info> changes the password of the specified user. (User must exist)')
+        ;
+    }
+
+    /**
+     * @return int|null|void
+     */
+    protected function serve()
+    {
+        $this->options = [
+            'user'        => $this->input->getOption('user'),
+            'password1'   => $this->input->getOption('password')
+        ];
+
+        $this->validateOptions();
+
+        $helper = $this->getHelper('question');
+        $data   = [];
+
+        $this->output->writeln('<green>Changing User Password</green>');
+        $this->output->writeln('');
+
+        if (!$this->options['user']) {
+            // Get username and validate
+            $question = new Question('Enter a <yellow>username</yellow>: ', 'admin');
```

Comment:
yep, i'll change that when i get back from dinner. would you also prefer a current password prompt if user being updated has full admin rights?